addons/sale_project/models/project_milestone.py

```diff
@@ -19,10 +19,3 @@ class ProjectMilestone(models.Model):
     @api.model
     def _get_fields_to_export(self):
         return super()._get_fields_to_export() + ['allow_billable', 'quantity_percentage', 'sale_line_name']
-
-    def action_get_list_view(self):
-        view = super().action_get_list_view()
-        project = self.env['project.project'].browse(self.env.context.get('active_id'))
-        if project.allow_billable:
-            view['views'] = [(self.env.ref('sale_project.project_milestone_view_tree').id, 'tree'), (False, 'form')]
-        return view
```

Comment:
You cannot remove the method you should deprecate it in stable and then in master you can remove it.

Reply:
As the parent method is deprecated we should be safe. It looks good to me. Thanks!